pxd.c

```diff
@@ -1132,7 +1132,7 @@ int pxd_init(void)
 out_misc:
 	misc_deregister(&pxd_miscdev);
 out_fuse:
-	for (j = 0; j < i; ++j) {
+	for (j = 0; j <= i; ++j) {
```

Comment:
Context at i need to be destroyed only if we are here from line: 1101. Else context at i is either uninitialized or beyond limit. You could just add a pxd_context_destroy() at line: 1101.